src/main/io/displayport_msp_bf_compat.c

```diff
@@ -117,14 +119,14 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_MAH_NM_1:
         return BF_SYM_MAH_NM_1;
-*/
+
     case SYM_MAH_KM_0:
-        return BF_SYM_MAH;
+        return BF_SYM_MAH_KM_0;
 
     case SYM_MAH_KM_1:
-        return 'K';     // K indicating Km
```

Comment:
Too vague. Not sold on replacing symbols with single letters.

Reply:
Only way I could think of to indicate Km. Agree it's not ideal, but much better than "??", wouldn't you agree?